modules/mw.cx.TargetArticle.js

```
@@ -15,7 +15,7 @@ mw.cx.TargetArticle = function MWCXTargetArticle(translation, veTarget, config)
 	this.veTarget = veTarget;
 	this.config = config;
 	this.siteMapper = config.siteMapper;
-	this.campaign = config.campaign;
+	this.campaign = config.campaign || '';;
```

ESLint

[error] Missing whitespace after semicolon.

(semi-spacing)